contrib/opencensus-ext-flask/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = '0.2.dev0'
```

Comment:
Do you still want to keep the versions numbers in sync?

Reply:
No, each package can have their independent version. I just inherited the current version, going forward it can be versioned/released separately from the core OpenCensus Python lib.
Do you think we should make it something like `0.1.dev0`?

Reply:
I think `0.1.dev0` makes sense.